src/constants/serviceAccount.js

```diff
@@ -0,0 +1,7 @@
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const serviceAccount = JSON.parse(process.env.SERVICE_ACCOUNT_KEY);
+
```

Comment:
is this file needed?

Reply:
Yes, is to get the access code to authenticate the backend